2010-08-04  Mark Mitchell  <mark@codesourcery.com>

	Backport from mainline:

	gcc/testsuite/
	2010-08-04  Daniel Gutson  <dgutson@codesourcery.com>
	* g++.dg/warn/miss-format-1.C: Update line number.

 2010-08-04  Julian Brown  <julian@codesourcery.com>
 
 	gcc/

=== modified file 'gcc/testsuite/g++.dg/warn/miss-format-1.C'
--- old/gcc/testsuite/g++.dg/warn/miss-format-1.C	2010-05-03 17:32:50 +0000
+++ new/gcc/testsuite/g++.dg/warn/miss-format-1.C	2010-08-20 15:08:29 +0000
@@ -4,7 +4,7 @@
 /* { dg-options "-Wmissing-format-attribute" } */
 /* { dg-options "-Wmissing-format-attribute -Wno-abi" { target arm_eabi } } */
 /* VxWorks does not provide vscanf, either in kernel or RTP mode.  */
-/* { dg-error "not declared" "" { target *-*-solaris2.[7-8] *-*-vxworks* } 25 } */
+/* { dg-error "not declared" "" { target *-*-solaris2.[7-8] *-*-vxworks* } 26 } */
 
 #include <stdio.h>
 #include <stdarg.h>

